Framework/SINQ/src/PoldiPeakSearch.cpp

```diff
@@ -583,7 +584,7 @@ void PoldiPeakSearch::exec() {
 
   g_log.information() << "   Parameters set.\n";
 
-  MantidVec summedNeighborCounts = getNeighborSums(correlatedCounts);
+  MantidVec summedNeighborCounts = getNeighborSums(correlatedCounts.rawData());
```

Comment:
using `const auto &summedNeighborCounts` doesn't change anything, check the implementation of `getNeighborSums()`.